libcodechecker/server/database/run_db_model.py

```diff
@@ -51,22 +51,42 @@ class Run(Base):
 
     id = Column(Integer, autoincrement=True, primary_key=True)
     date = Column(DateTime)
-    duration = Column(Integer)  # Seconds, -1 if unfinished.
+
+    # If positive: the seconds the duration of the run.
+    # If -1, the run is undergoing a store, but there exists a previous state.
+    # If -2, the run is undergoing its first store.
+    duration = Column(Integer)
```

Comment:
Wouldn't it be better to create a separate column for this instead of playing with negative numbers?

Reply:
It has been explained earlier, read the pull requests' text.

Creating a separate column would be really, really heavy on the database. The negative value is automatically eliminated once the run's first store action concludes, whereas a boolean value would store a useless dummy `TRUE` (or `FALSE`) for each and every run indefinitely.